.drone.star

```diff
@@ -93,7 +93,7 @@ config = {
                 "skip": False,
             },
             "ocis": {
-                "version": "5.0.0",
+                "version": "6.6.0",
```

Comment:
I think, it's better to use the production version. So, I will wait till we get the production version tag
```suggestion
                "version": "7.0.0",
```